src/components/dialogs/script-dialog.js

```diff
@@ -169,7 +169,9 @@ const ScriptDialog = ({ id, open, onClose, onError, title, type }) => {
 
     useEffect(() => {
         // get contingency list
-        getCurrentScript(id);
+        if (id !== '') {
```

Comment:
needed this change to fix a bug due to change of initialisation of the listId variable. Not needed anymore since it was changed in another PR. Removed.